addons/account/models/account_move.py

```diff
@@ -1716,6 +1717,15 @@ def _inverse_name(self):
             move.name and move.name != '/'
         ))
 
+    def _inverse_fiscal_position_id(self):
+        # _origin is not working in inverse so we need fatched data from database to check old value.
+        self.env.cr.execute("SELECT id,fiscal_position_id FROM account_move WHERE id in %s", [tuple(self.ids)])
+        old_fiscal_position_id = dict(self.env.cr.fetchall())
+        for move in self:
+            if (old_fiscal_position_id.get(move.id) or move.fiscal_position_id) and move.state == 'draft':
```

Comment:
So this is checking if either
* there was a FP before
* there is a FP now

Is there any use case where we would enter `_inverse_fiscal_position_id` but there isn't at least one of those cases that is true?

Reply:
Could you please provide more details? 
I didn't fully understand what you were trying to say.

Reply:
When do we enter this function (`_inverse_fiscal_position_id`) in practice, and we don't want to apply the mapping of fiscal positions?

Reply:
@william-andre inverse also calls on create and at that time doesn't want to apply the mapping
normally `inverse` is for the non-store field but for this store field and still has inverse so this happens.
so there is 3 ways to handle it.

1. check for old value
2. fix on models to not call inverse from create when the field is stored
3. do this with `api.onchange` (no mapping if writing from backend)